server/api/user/index.js

```diff
@@ -33,5 +33,7 @@ router.delete('/:id/project', auth.isAuthenticated(), controller.removeProject);
 router.put("/:id/addTech", auth.canEdit(), controller.addTech);
 router.put("/:id/removeTech", auth.canEdit(), controller.removeTech);
 router.post('/resetPassword', controller.resetPassword);
+router.put('/:id/removeUser',controller.deleteUser); 
+router.put('/:id/deactUser',auth.canEdit(),controller.deactivate);
```

Comment:
we already have a deactivate route. See '/:id/deactivate' i think this is from a merge conflict you tried resolving.